plaidml2/bridge/keras/__init__.py

```diff
@@ -40,6 +44,24 @@ def _prepend_name_scope(name, default):
     return r
 
 
+def _normalize_axis(axis, ndims, name=""):
```

Comment:
There's some inconsistency in this file between double-quotes and single-quotes.  Please use single-quotes for all strings except doc comments.